### ISSUE TITLE
为什么我的valObjectness 先下降后上升

### ISSUE THREAD

**Issue by XIEdeda** [ultralytics/yolov5#2528]
## Additional context
![image](https://user-images.githubusercontent.com/68369130/111777763-ae222780-88ee-11eb-9894-2b9205ef5ac2.png)

**Comment by XIEdeda** [comment]
我怎么样才能使我的训练变得更好

**Comment by glenn-jocher** [comment]
@XIEdeda your objectness loss is overfitting. You should reduce your obj or obj_pw hyperparameter to reduce objectness loss contribution to your total loss, especially as your obj val loss magnitude is much higher than it's neighbors.

**Comment by glenn-jocher** [comment]
@XIEdeda the place to do this is here. You can also try using --hyp hyp.finetune.yaml rather than hyp.scratch.yaml.
https://github.com/ultralytics/yolov5/blob/ed2c74218d6d46605cc5fa68ce9bd6ece213abe4/data/hyp.scratch.yaml#L16-L17

**Comment by XIEdeda** [comment]
@glenn-jocher I used it-- hyp.finetune But obj Val loss is still the same effect. What should I do？

**Comment by glenn-jocher** [comment]
@XIEdeda you can try reducing the objectness hyperparameter values here:
https://github.com/ultralytics/yolov5/blob/ed2c74218d6d46605cc5fa68ce9bd6ece213abe4/data/hyp.scratch.yaml#L16-L17

**Comment by XIEdeda** [comment]
@glenn-jocher ok thank you!  i will try it

**Comment by yums24** [comment]
> @glenn-jocher ok thank you! i will try it
hi, I also have the same problem, may I ask how to adjus,
just reduce the value of obj parameter？

**Comment by glenn-jocher** [comment]
@yums24 you can try reducing the objectness hyperparameter values here:
https://github.com/ultralytics/yolov5/blob/ed2c74218d6d46605cc5fa68ce9bd6ece213abe4/data/hyp.scratch.yaml#L16-L17
